sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -683,6 +712,7 @@ public void run() {
 
     @WorkerThread
     private synchronized void handleCheckForUpdate() {
+        mManualCheckForUpdateRequested = true;
```

Comment:
You need to reset it to false when the workflowCompleted is updated to true and a need unit test around that: restarting launcher activity after having done a successful manual check for update must do another check until called again explicitly.

Reply:
Added the code to reset.